Assets/MixedRealityToolkit.Providers/ObjectMeshObserver/SpatialObjectMeshObserverProfile.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.SpatialAwareness;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.SpatialObjectMeshObserver
+{
+    /// <summary>
+    /// Configuration profile for the spatial object mesh observer.
+    /// </summary>
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Object Mesh Observer Profile", fileName = "ObjectMeshObserverProfile", order = 1000)] // todo
```

Comment:
>// todo

Still todo?

Reply:
it was todo to pick the right number :\

Reply:
removed

Reply:
Should this be added to the `ProfileMenuItemIndices` enum?

Reply:
it feels like we need a followup PR to separate service and data provider menus.